libcst/codemod/visitors/_apply_type_annotations.py

```diff
@@ -697,6 +851,54 @@ def transform_module_impl(
         else:
             return tree
 
+    # helpers for collecting type information from the stub files
+
+    def _get_module_imports(
+        self, stub: cst.Module, existing_import_names: Set[str]
+    ) -> Dict[str, ImportItem]:
+        """Returns a dict of modules that need to be imported to qualify symbols."""
+        # We correlate all imported symbols, e.g. foo.bar.Baz, with a list of module
+        # and from imports. If the same unqualified symbol is used from different
+        # modules, we give preference to an explicit from-import if any, and qualify
+        # everything else by importing the module.
+        #
+        # e.g. the following stub:
+        #   import foo as quux
+        #   from bar import Baz as X
+        #   def f(x: X) -> quux.X: ...
+        # will return {'foo': ImportItem("foo", "quux")}. When the apply type
+        # annotation visitor hits `quux.X` it will retrieve the canonical name
+        # `foo.X` and then note that `foo` is in the module imports map, so it will
+        # leave the symbol qualified.
+        import_gatherer = GatherImportsVisitor(CodemodContext())
+        stub.visit(import_gatherer)
+        symbol_map = import_gatherer.symbol_mapping
+        isc = ImportedSymbolCollector(existing_import_names, self.context)
+        cst.MetadataWrapper(stub).visit(isc)
+        module_imports = {}
+        for sym, isyms in isc.imported_symbols.items():
```

Comment:
What does `isyms` refer to?

I'm not sure about LibCST, but on pyre we're generally anti abbreviations; I personally don't mind some acronyms like `sym` too much but the combination of `isc`, `sym`, `isym`, and `imp` starts to be a bit alphabet-soup-y

Reply:
done